assets/templates/dataset-filter/hierarchy.tmpl

```diff
@@ -68,8 +68,10 @@
 								</div>
 							{{else}}
 								<input name="save-and-return" class="visuallyhidden" type="submit" value="Save and return" />
-								<input class="btn btn--link underline-link js-filter add-all" type="submit" value="Add all" name="add-all" />&nbsp; &nbsp;
+								<input class="btn btn--link underline-link js-filter add-all" type="submit" value="Add all" name="add-all" id="add-all"/>
+								<label for="add-all"><span class="visuallyhidden">{{.Data.DimensionName}} to list</span></label>&nbsp; &nbsp;
```

Comment:
span is unnecessary and using the Mac accessibility VoiceOver I get “Geography to list button” gap “add-all. You are currently on a button” so “Add all {{.Data.DimensionName}}’s to list” and “Remove all {{.Data.DimensionName}}’s from list” might be better.